### PR TITLE
Refactored DOLDatabase to CoreDatabase, standardized DB class names

### PR DESCRIPTION
**Pull request by deftness02**

### CODE REVIEW

File: CoreDatabase/Tables/DBHousePermissions.cs
Comment (bm01) [inline CoreDatabase/Tables/DBHousePermissions.cs:1]: Inconsistent file name capitalization

File: DOLServer/config/serverconfig.example.xml
Comment (bm01) [inline DOLServer/config/serverconfig.example.xml:1]: This probably shouldn't be deleted

**Review by bm01 — CHANGES_REQUESTED** [review]
You pluralized DB class names but they all represent a single object. Each GameNPC has its own DbMob for example.

File: DOLServer/config/serverconfig.example.xml
Comment (deftness02) [reply DOLServer/config/serverconfig.example.xml:1]: Phooey, staged that change incorrectly.

**Comment by deftness02** [comment]
> You pluralized DB class names but they all represent a single object.

I was viewing this change more in mind as the class name representing the table object itself since the class name identifies the table prior to looking up any contained data, which I then pluralized in consideration for a table containing (typically) multiple records. But that was just my thought process, I can turn it singular if preferred.

**Comment by bm01** [comment]
Yeah I think it would be more consistent to keep them singular. What ties those classes with the DB tables is really just the attributes, but each instance represents one line.

**Pull request closed without merging**